docker/api/client.py

```diff
@@ -103,7 +103,8 @@ class APIClient(
     def __init__(self, base_url=None, version=None,
                  timeout=DEFAULT_TIMEOUT_SECONDS, tls=False,
                  user_agent=DEFAULT_USER_AGENT, num_pools=None,
-                 credstore_env=None, use_ssh_client=False):
+                 max_pool_size=DEFAULT_MAX_POOL_SIZE, credstore_env=None,
+                 use_ssh_client=False):
```

Comment:
I would move the newest parameter to the end of the list so that if users are using this constructer with positional parameters, it will continue to work as expected.
```suggestion
                 credstore_env=None, use_ssh_client=False,
                 max_pool_size=DEFAULT_MAX_POOL_SIZE):
```

Reply:
@chris-crone, I did it. Thanks for the suggestion! :smile: